cmd/kubeadm/app/preflight/checks.go

```diff
@@ -325,6 +337,112 @@ func (sysver SystemVerificationCheck) Check() (warnings, errors []error) {
 	return nil, nil
 }
 
+type etcdVersionResponse struct {
```

Comment:
I'd put the json serialization tags here to make it clear that this is a just a chunk of data that we get from etcd.